binary_transparency/firmware/internal/verify/bundle_test.go

```diff
@@ -50,30 +53,38 @@ func TestBundleForUpdate(t *testing.T) {
 	} {
 		t.Run(test.desc, func(t *testing.T) {
 			imgHash := sha512.Sum512(test.img)
-			err := verify.BundleForUpdate(up.ProofBundle, imgHash[:])
+			err := verify.BundleForUpdate(up.ProofBundle, imgHash[:], dc, proof)
 			if (err != nil) != test.wantErr {
 				t.Fatalf("want err %T, got %q", test.wantErr, err)
 			}
 		})
 	}
 }
 
+func b64Decode(t *testing.T, b64 string) []byte {
+	t.Helper()
+	st, err := base64.StdEncoding.DecodeString(b64)
+	if err != nil {
+		t.Fatalf("b64 decoding failed: %v", err)
+	}
+	return st
+}
+
 func TestBundleForBoot(t *testing.T) {
 	var up api.UpdatePackage
 	if err := json.Unmarshal([]byte(goldenUpdate), &up); err != nil {
 		t.Fatalf(err.Error())
 	}
-	imgHash := sha512.Sum512([]byte(goldenFirmwareImage))
 
+	b64 := "6tUWyykzwfoonGugnf1dL+cgd6bEWUholANaDL8JYKWtdmPcEeYvgO6nMr/ILgWPcUYggvAArrtaSBman5H4Kg=="
```

Comment:
Could you add a comment on where this b64 came from and how it should be updated if the goldenUpdate data is changed?

(also it should probably be up in `const` next to the `goldenUpdate`)